src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
   // write your code here
+  const stylesArray = sourceString
+    .split(';')
+    .map((item) => item.split(':').map((part) => part.trim()))
+    .filter((item) => item.length === 2 && item[0] && item[1]);
```

Comment:
The filter condition `item.length === 2 && item[0] && item[1]` is a good way to ensure that only valid CSS declarations are included. However, consider handling the case where the CSS property or value might contain ':' which could lead to incorrect splitting.